wolfssl/wolfcrypt/memory.h

```diff
@@ -95,12 +95,12 @@ WOLFSSL_API int wolfSSL_GetAllocators(wolfSSL_Malloc_cb*,
         #define WOLFMEM_IO_SZ        16992 /* 16 byte aligned */
     #endif
     #ifndef WOLFMEM_BUCKETS
-        /* default size of chunks of memory to seperate into
-         * having session certs enabled makes a 21k SSL struct */
+        /* default size of chunks of memory to separate into
+         * having session certs enabled makes a 24k SSL struct */
```

Comment:
Can you describe why this increase was required? If its just for the `WOLFSSL_CERT_EXT` case maybe we should only increase it for that build case.

Reply:
Got it

Reply:
The increased memory requirement was due to the addition of the raw subject and issuer fields in the cert  structure, and the newly supported OIDs in the certName structure.